code/game/gamemodes/cult/runes.dm

```diff
@@ -437,29 +437,30 @@ var/list/sacrificed = list()
 /////////////////////////////////////////ELEVENTH RUNE
 
 /obj/effect/rune/proc/manifest(var/mob/living/user)
-	var/obj/effect/rune/this_rune = src
-	if(user.loc!=this_rune.loc)
-		return this_rune.fizzle(user)
+	if(user.loc!=src.loc)
```

Comment:
```suggestion
	if(user.loc!=loc)
```

Isn't necessary to have the `src` here (and elsewhere) as the replacement for `this_rune`.

Reply:
It's honestly a preference and really has no performance loss. It makes the code easier to read when comparing two different locs.